bazel-haskell-deps.bzl

```diff
@@ -457,15 +479,8 @@ exports_files(["stack.exe"], visibility = ["//visibility:public"])
                 "hlint": ["ghc-lib"],
                 "shake": ["embed-files"],
                 "zip": ["disable-bzip2", "disable-zstd"],
+                "proto3-suite": ["-swagger"],
```

Comment:
I'm curious, why do we need to specify this flag here if it's already part of our vendored `proto3-suite`?

Reply:
Because otherwise `bazel run @stackage-unpinned//:pin` doesn’t seem to pick up that flag (the actual build later does) which means we still need to have swagger2 somewhere. Not quite sure why that is happening, I’ll talk to Andreas on Thursday to ask.

Reply:
oh interesting. thanks for explaining!